### DISCUSSION TITLE
BTT E3 RRF v1 & TFT35 E3 v3.0 "No Printer Attached"

### DISCUSSION THREAD

**Discussion by 3DPrintJunkee** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1994] [Q&A]
I recently installed a new E3 RRF control board and TFT35 E3 V3.0 display. Flashed firmware for both. When I turn the printer on I get a "No Printer Attached". I've made sure the Baudrates match in both the control board and display firmware. I tried changing the serial ports in the control board firmware to 0 and -1 respectively and the firmware won't compile. Not sure what else to try. Every YT video I've seen the problem either goes away after 10-15 seconds or the baudrates weren't aligned. Using both the grey ribbon (EXP1) and the 5 pin cable (RS232)

Any advice/suggestions/feedback would be greatly appreciated.

Display firmware is BIGTREE_TFT35_V3.0_E3.27.x.bin

**Comment by kisslorand** [comment]
Just today happened to me also . "reset.txt" on the SD card root folder helped.

**Reply by 3DPrintJunkee** [reply]
"reset.txt"? What is in the text file?

**Reply by 3DPrintJunkee** [reply]
Tried the reset.txt in the SD card root. Didn't fix it.

**Comment by tome9111991** [comment]
I have same issue with E3 Turbo

**Reply by tome9111991** [reply]
I solved my issue.
I have to use RS232 and tft port

**Reply by 3DPrintJunkee** [reply]
Glad you resolved it. I am still no further ahead. I used the firmware from github. Still getting "No Printer Attached" except it will allow my to select Marlin Mode but nothing happens. Just displays Marlin Mode across the top of the screen.

**Reply by jobobeda** [reply]
Same error "no printer attached".  No response going into "Marlin Mode",  Touch buttons will act but "no Printer Attached" keeps anything from happening.  I'm using Configure.h and Configure_adv.h from Gamster  downloaded from Gethub.com  configurations.   There must be some other settings I'm missing, both baudrates match and have been changed several times (matching each time).  What the heck is missing?  It shouldn't be this difficult to program Marlin 2.0 for a CR10 printer with an SKR-2 board!  By the numbers instructions from BTT would be a Godsend, in fact any support from BTT would be appreciated!  How did others solve this problem?
